bash_command_line_101/bash_command_line_101.md

```diff
@@ -374,23 +398,68 @@ This is another sentence.
 **Give it a try!** Add some text to the files you already created!
 
 <div class = "warning">
+<b style="color: rgb(var(--color-highlight));">Warning!</b><br>
+
 Make sure you use the two greater than signs (here, we'll call them "arrows") in `>>`. Using only a single arrow will also write text to your file, but instead of appending it to the end of the file, it will replace the entire contents of the file with that new text.
+
 </div>
 
 Next, we will see learn how to look at the contents of our file to ensure that it really does contain what we think it should.
 
 ### Viewing the contents of a file
 
-The easiest way to see the contents of a file is the `cat` function. If you run `cat` followed by a file name, the command line interface will print out the contents of that file line by line.
+There are several different ways to view the contents of a file. You might want to see every line of a short file printed in its entirety, but if your file is 1000 lines long, or has lines containing thousands of characters, you might want to look at particular parts of the file.
+
+`cat`
+---
+The easiest way to see the contents of a short file is the `cat` function. If you run `cat` followed by a file name, the command line interface will print out the entire contents of that file line by line.
 
-Give it a try:
+**Give it a try:**
 
 ```
 cat my_sentences
 ```
 
 <div class = "learn-more">
+<b style="color: rgb(var(--color-highlight));">Learning connection</b><br>
+
 The command `cat` is very powerful three-part function that allows a reader to view, combine, or create a new version of a file. In fact `cat` is a shortening of the word "con**cat**enate."
```

Comment:
I think this could be a good place to link to the manual entry for `cat`, since it does more things than we actually talk about here.